src/main/java/geekbrains/java2/ht2/Board.java

```diff
@@ -0,0 +1,111 @@
+package geekbrains.java2.ht2;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+class Board {
+
+    final static int SIZE = getBoardSize();
+    private final static char EMPTY = '*';
+    final static char USER = 'X' ;
+    final static char COMP = 'O' ;
+    static char [][] board = new char[SIZE][SIZE] ;
```

Comment:
\[Checkstyle\] ERROR: Variable 'board' must be private and have accessor methods\.